documentation/onboard-form-template.md

```diff
@@ -0,0 +1,34 @@
+# Teacher services infra - Service onboarding form
+- Service name? e.g. calculate-teacher-pay
+- Domains  ? e.g. school-placements-production.teacherservices.cloud
+- Service short name? e.g. ctp
+- Repository? e.g. https://github.com/DFE-Digital/itt-mentor-services
+- Environments? e.g. develoment, staging, production
+- Review apps? Yes or No
+- Cost centre? e.g. 10168
+- Activity code? e.g
+    - placements: 101423
+    - track: 101056
+- List of developers who will need access to Azure? Do they use a DfE laptop or BYOD? e.g
+    - James Beagle - BOYD
+    - Clinton Morris - DfE laptop
+- GDS assessed? Yes or No
+
+- Slack channels? e.g
+    - Product: #itt-mentor-services
+    - Tech: #itt-mentor-services-devs
+- Timeline? e.g
+    - QA/Staging ASAP
+    - Prod track-and-pay April
+    - Prod school-placements TBC but likely April
+- Github repos / Docker registry type: open
+- Namespaces e.g
+    - bat-qa
+    - bat-staging
+    - bat-production
+- Technical requirements
+    - Ruby on rails webapp
+    - postgres
+    - Future: worker and redis for sidekiq
+- Healthcheck page
+-
```

Comment:
Can we have a sample healthcheck endpoint?

Reply:
It should be /healthcheck by default as per https://github.com/DFE-Digital/terraform-modules/blob/main/aks/application/variables.tf#L104